docs/user/document/tracking/aliases-usage.en.md

```diff
@@ -1 +1,8 @@
 # Aliases - Usage
+
+Aliases can be used to indicate the number assigned to this advisory by the coordinator or manufacturer.
```

Comment:
```suggestion
Aliases can be used to indicate the number, identifier or name assigned to this advisory, case or issue by the coordinator, manufacturer, supplier or other party. This can also include internal ticket numbers.